### PR TITLE
update webhook to use API key

### PR DESCRIPTION
**Pull request by npmenard**

### CODE REVIEW

File: src/esp32/webhook.rs [hunk 1]
```diff
@@ -40,8 +40,10 @@ pub struct Webhook {
     fqdn: String,
     /// Endpoint that will be sent a GET request with credential information
     endpoint: Option<String>,
-    /// Location Secret used by the webhook's SDK script to connect
-    secret: Option<String>,
+    /// API key ID used by the webhook's SDK script to connect
+    api_key_id: Option<String>,
+    /// API key used by the webhook's SDK script to connect
+    api_key: Option<String>,
```
Comment (randhid) [inline src/esp32/webhook.rs:46]: uhhhh I think fleet isn't doing this anymore. Can you check with emily and roxy?

Comment (randhid) [reply src/esp32/webhook.rs:46]: unless this is the only secret that fleet 2 is exposing.

Comment (npmenard) [reply src/esp32/webhook.rs:46]: i mean this is what the example code show since a couple a days. they are removing location secret

Comment (randhid) [reply src/esp32/webhook.rs:46]: cool - I'm probably mistaken then.

**Pull request merged**